rmw_connext_shared_cpp/src/node.cpp

```diff
@@ -436,74 +468,64 @@ create_node(
 rmw_ret_t
 destroy_node(const char * implementation_identifier, rmw_node_t * node)
 {
-  if (!node) {
-    RMW_SET_ERROR_MSG("node handle is null");
-    return RMW_RET_ERROR;
-  }
+  assert(implementation_identifier != NULL);
```

Comment:
Same comment about checking and returning on NULL, and using `nullptr` instead.

Reply:
I thought about it, but this `rmw_connext*` packages are already inconsistent in this regard pretty much everywhere. If you agree, I'd prefer doing that in a separate PR.